generators/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -170,23 +229,24 @@ function installChannels() {
 function networkDown() {
   printHeadline "Destroying network" "U1F916"
   (
-    cd "$FABRICA_NETWORK_ROOT"/fabric-docker
-    docker-compose down
+    cd "$FABRICA_NETWORK_ROOT"/fabric-docker && \
+      docker-compose down
   )
 
   printf "\nRemoving chaincode containers & images... \U1F5D1 \n"
-   <% chaincodes.forEach(function(chaincode) {
-     chaincode.channel.orgs.forEach(function (org) {
-       org.peers.forEach(function (peer) {
-        var chaincodeContainerName="dev-"+peer.address+"-"+chaincode.name+"-"+chaincode.version
-  %>
-  docker rm -f $(docker ps -a | grep <%= chaincodeContainerName %>-* | awk '{print $1}') || {
-    echo "docker rm failed, Check if all fabric dockers properly was deleted"
-  }
-  docker rmi $(docker images <%= chaincodeContainerName %>-* -q) || {
-    echo "docker rm failed, Check if all fabric dockers properly was deleted"
-  }
-  <% })})}) -%>
+  <% chaincodes.forEach(function(chaincode) { -%>
+    <% chaincode.channel.orgs.forEach(function (org) { -%>
+      <% org.peers.forEach(function (peer) { -%>
+        <% const chaincodeContainerName="dev-"+peer.address+"-"+chaincode.name+"-"+chaincode.version -%>
+        docker rm -f $(docker ps -a | grep <%= chaincodeContainerName %>-* | awk '{print $1}') || {
+          echo "docker rm failed, Check if all fabric dockers properly was deleted"
+        }
```

Comment:
I know that it's pretty old code, but... all of docker commands besides this one are wrapped into much shorter utility scripts. maybe we could wrap this one also?